wrappers/services.go

```diff
@@ -204,6 +205,17 @@ WantedBy={{.ServicesTarget}}
 		restartCond = systemd.RestartOnFailure.String()
 	}
 
+	var remain string
+	if appInfo.Daemon == "oneshot" {
```

Comment:
Just thinking aloud. Is there any other condition where we would like to have remain-after-exit set?